lib/rack/request.rb

```diff
@@ -482,9 +482,14 @@ def parseable_data?
 
       # Returns the data received in the query string.
       def GET
-        if get_header(RACK_REQUEST_QUERY_STRING) == query_string
+        rr_query_string = get_header(RACK_REQUEST_QUERY_STRING)
```

Comment:
Is it possible to think of a better name than `rr_query_string`? maybe `cached_query_string` or `parsed_query_string` or `rack_request_query_string` (I know, kind of long).

Reply:
Probably, but considering this is going to be removed in Rack 3.2, I'm not sure it's worth coming up with a better name.  Are you OK with me merging as-is?

Reply:
Sure :)